src/coreclr/src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -124,7 +124,7 @@ public static object GetClassFactoryForType(ComActivationContext cxt)
 
             if (!Path.IsPathRooted(cxt.AssemblyPath))
             {
-                throw new ArgumentException();
+                throw new ArgumentException(null, nameof(cxt));
```

Comment:
This API is only called from a COM activation scenario and as such none of these details will be exposed to the caller - only the associated `HRESULT`. I'm no opposed to adding them, but there is no user impact here.

Reply:
Sounds like keeping it not hurt, I would like to keep it to avoid the warning triggered